openedx/frontend-app-course-authoring
# feat: bare bones taxonomy detail page [FC-0036]

## Description
Add the first version of the Taxonomy Detail page to show taxonomy metadata and its tag list.

## Supporting Information
* Closes https://github.com/openedx/modular-learning/issues/138
* Depends on:
  *  https://github.com/openedx/frontend-app-course-authoring/pull/645

![image](https://github.com/openedx/frontend-app-course-authoring/assets/849463/90b7ff59-ea62-4416-8ea0-1f87e9c838b5)

## Testing instructions
- Go to the taxonomy list page (http://localhost:2001/taxonomy-list)
- Click in any place of the taxonomy card to go to the taxonomy detail page
- Verify if the tag list is displayed and the taxonomy info is showed in the right card
- Use the `Actions` menu to export the taxonomy (taxonomy modal implemented [here](https://github.com/openedx/frontend-app-course-authoring/pull/645))

If you don't have any taxonomies registered, you could use the  [taxonomy-sample-data](https://github.com/open-craft/taxonomy-sample-data/tree/main) script.
____
Private-ref: [FAL-3529](https://tasks.opencraft.com/browse/FAL-3529)
